pkg/services/sqlstore/sqlstore.go

```diff
@@ -179,6 +181,20 @@ func (ss *SQLStore) GetDialect() migrator.Dialect {
 	return ss.Dialect
 }
 
+func (ss *SQLStore) GetDB() *sqlx.DB {
+	if ss.sqlxdb == nil {
+		ss.sqlxdb = sqlx.NewDb(ss.engine.DB().DB, ss.GetDialect().DriverName())
+	}
+	return ss.sqlxdb
+}
+
+func (ss *SQLStore) BuildInsertWithReturningId(query string) (string, bool) {
+	if ss.Dialect.DriverName() == "postgres" {
```

Comment:
I think that this is supposed to be used only by `sqlx` so I would suggest the following change
so that we can remove it from the [DB](https://github.com/grafana/grafana/blob/50fe9b764d99fb57dc27bc8a40842bd322f8fc28/pkg/services/sqlstore/db/db.go#L11) interface and use it only internally by `sqlx`
```suggestion
	if ss.sqlxdb.DriverName() == "postgres" {
```